variables.tf

```diff
@@ -1,24 +1,259 @@
+variable "satellite_host_count" {
+  description = "The total number of AWS host to create for control plane. satellite_host_count value should always be in multiples of 3, such as 3, 6, 9, or 12 hosts"
+  type        = number
+  default     = 3
+  validation {
+    condition     = (var.satellite_host_count % 3) == 0 && var.satellite_host_count > 0
+    error_message = "Sorry, host_count value should always be in multiples of 3, such as 6, 9, or 12 hosts."
+  }
+}
+
+variable "addl_host_count" {
+  description = "The total number of additional aws host"
+  type        = number
+  default     = 3
+}
+
+variable "location_instance_type" {
+  description = "The type of aws instance to create"
+  type        = string
+  default     = "m5d.xlarge"
+}
+
+variable "cluster_instance_type" {
+  description = "The type of aws instance to create"
+  type        = string
+  default     = "m5d.xlarge"
+}
+
+variable "ssh_public_key" {
+  description = "SSH Public Key. Get your ssh key by running `ssh-key-gen` command"
+  type        = string
+  default     = null
+}
+
+variable "resource_prefix" {
+  description = "Name to be used on all aws resource as prefix"
+  type        = string
+  default     = "satellite-aws"
+
+  validation {
+    condition     = var.resource_prefix != "" && length(var.resource_prefix) <= 25
+    error_message = "Sorry, please provide value for resource_prefix variable or check the length of resource_prefix it should be less than 25 chars."
+  }
+}
+
+##################################################
+# IBMCLOUD ROKS Cluster Variables
+##################################################
+
+variable "create_cluster" {
+  description = "Create Cluster"
+  type        = bool
+  default     = true
+}
+
+variable "cluster" {
+  description = "Cluster name"
+  type        = string
+  default     = "satellite-ibm-cluster"
+
+  validation {
+    error_message = "Cluster name must begin and end with a letter and contain only letters, numbers, and - characters."
+    condition     = can(regex("^([a-z]|[a-z][-a-z0-9]*[a-z0-9])$", var.cluster))
+  }
+}
+
+variable "kube_version" {
+  description = "Satellite Kube Version"
+  default     = "4.7_openshift"
+}
+
+variable "cluster_host_labels" {
+  description = "Labels to add to attach host script"
+  type        = list(string)
+  default     = ["env:prod"]
+
+  validation {
+    condition     = can([for s in var.cluster_host_labels : regex("^[a-zA-Z0-9:]+$", s)])
+    error_message = "Label must be of the form `key:value`."
+  }
+}
+
+variable "worker_count" {
+  description = "Worker Count for default pool"
+  type        = number
+  default     = 1
+}
+
+variable "wait_for_worker_update" {
+  description = "Wait for worker update"
   type        = bool
   default     = true
 }
 
-********************************************************************/
\ No newline at end of file
+variable "default_worker_pool_labels" {
+  description = "Label to add default worker pool"
+  type        = map(any)
```

Comment:
Please define the contents of default_worker_pool_labels. When use sees map(any) he will not be sure what to pass

Reply:
added input.tfvars file and defined default_worker_pool_labels example for usage